### DISCUSSION TITLE
Type 'ReactElement<any, string | JSXElementConstructor<any>>' is not assignable to type 'ReactNode'.

### DISCUSSION THREAD

**Discussion by okhaimie-dev** [shadcn-ui/ui#1322] [Q&A]
Guys, I am having issues with my button and almost all my components imported from `shadcn/ui`. I am assuming this might have something to do with my `@types/react` package, but I am not sure what to downgrade it too.

Does anyone have an idea how I could go about debugging this issue? I have looked around the internet, and I cannot seem to find a fix.

![CleanShot 2023-08-22 at 22 41 06](https://github.com/shadcn-ui/ui/assets/57156589/93e20c89-53be-4de5-a79c-49bfc2b54f1d)

**Comment by okhaimie-dev** [comment]
This is a similar issue, but suggestions doesn't seem to work. https://github.com/shadcn-ui/ui/issues/1103

I have installed the exact versions suggested here in

**Reply by shrix1** [reply]
Detailed code please or repo link please ?

**Comment by okhaimie-dev** [comment]
Here is a link to the code base.
https://github.com/DAObox/tribeOs/blob/develop/apps/web/src/components/ui/button.tsx

& `package.json`
https://github.com/DAObox/tribeOs/blob/develop/apps/web/package.json

Thanks

**Reply by shrix1** [reply]
Did you tried upgrading packages to latest ??

**Reply by okhaimie-dev** [reply]
Yes, I did. Also tried to match the versions recommended in a similar issue.

Didn't help

**Reply by shrix1** [reply]
okay , will see
